src/commctl/cli.py

```diff
@@ -544,6 +570,23 @@ def add_cluster_commands(argument_parser):
     subject_subparser.add_parser('list')
     # No arguments for 'cluster list' at present.
 
+    # Command: cluster deploy ...
+
+    object_parser = subject_subparser.add_parser('deploy')
+    object_subparser = object_parser.add_subparsers(dest='subcommand')
+    object_subparser.required = True
+
+    # Sub-command: cluster deploy start
+    verb_parser = object_subparser.add_parser('start')
+    verb_parser.required = True
+    verb_parser.add_argument('name', help='Name of the cluster')
+    verb_parser.add_argument('version', help='Version to deploy')
```

Comment:
Not sure I understand.  The `version` argument *is* required.  Or are you talking package dependencies?